client/ayon_blender/plugins/create/create_blendScene.py

```diff
@@ -40,3 +45,12 @@ def remove_instances(self, instances):
             bpy.data.collections.remove(node)
 
             self._remove_instance_from_context(instance)
+
+    def get_pre_create_attr_defs(self):
+        defs = super().get_pre_create_attr_defs()
+        defs.extend([
+            BoolDef("selected_hierarchies",
+                    label="Select Hierarchies",
+                    default=False)
+        ])
```

Comment:
Select hierarchies is confusingly named - it doesn't select anything.

- Include hierarchy
- Include all children   (this may be more accurate, because we're not actually including parent hierarchy?)
- Link all recursive children   (this may be more accurate in blender terminology)
- Link all recursive children of objects   (this may be most accurate, because we might not want to do it on Collections?)

Anyway, this 'fixes' what's linked in the Collection solely for `CreateBlendScene` but the issue is also present on `CreatePointcache`.